### PR TITLE
Update state diagrams

### PR DESCRIPTION
**Pull request by tomclose**
This PR updates the state diagrams in the readme to reflect the work that will be part of the upcoming sprint (by stripping out the direct funding parts). It also scaffolds out the directory structure for the `player-a / player-b` state machines.

### CODE REVIEW

File: packages/wallet/src/redux/indirect-funding/readme.md [hunk 1]
```diff
@@ -116,24 +115,19 @@ Once the state machine reaches a terminated state, the process is dismantled.
...
 ```
 
 ### Player B
 
 ```mermaid
   graph TD
   linkStyle default interpolate basis
-    WAIT_FOR_STRATEGY_REQUEST -->|INDIRECT| indirectStrategy[WAIT_FOR_APPROVAL]
-    WAIT_FOR_STRATEGY_REQUEST -->|DIRECT| directStrategy[WAIT_FOR_APPROVAL]
+    WAIT_FOR_APPROVAL -->|FUNDING_APPROVED| WAIT_FOR_STRATEGY_REQUEST
```
[the remark this answers is not in the document]
Comment (tomclose) [reply packages/wallet/src/redux/indirect-funding/readme.md]: I didn’t mean to add a state - I just moved it. Both players start by approving that they want the channel funded. A then proposes the indirect strategy and B approves the choice of strategy. The previous diagram had B waiting for the strategy before they had approved the funding. My intention was to change the order on that.

[the remark this answers is not in the document]
Comment (andrewgordstewart) [reply packages/wallet/src/redux/indirect-funding/readme.md]: In the old diagram, player b's wallet waits for the strategy request from player A's wallet. It does not wait for player A to approve a strategy.

In the new diagram, it seems like you're saying:
- Player B approves funding, which is implicitly indirect funding
- Player B then approves funding again
and there is no response to a `STRATEGY_PROPOSED` message from player a's wallet.

If something here is incorrect, what is it?

[the remark this answers is not in the document]
Comment (tomclose) [reply packages/wallet/src/redux/indirect-funding/readme.md]: The thing that isn't correct here is that it isn't Player B who then approves funding again - it's Player B's wallet that approves the strategy, as the response to Player A's `STRATEGY_PROPOSED`.

**Comment by tomclose** [comment]
Close this because it's now so stale it isn't worth trying to merge ...

File: packages/wallet/src/redux/indirect-funding/readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewgordstewart) [reply packages/wallet/src/redux/indirect-funding/readme.md]: Ok, it looks like you've clarified what the action is? 👍

**Pull request closed without merging**